src/Session/commands.py

```diff
@@ -79,10 +79,38 @@ def list_freezers(self):
 
     def load(self, profile):
         pass
-    
-    def open(self, food_storage):
-        pass
+  
+    ''' Prompts user input to open a food storage. Also lists available food storages user can choose from. 
+        If a food storage with that name does not exist, prompts user to create a new one. '''
+    def open(self, foodStorage_type):
+        foodStorage_list = self.profile.listFoodStorages(foodStorage_type)
+        print(OPEN_FOOD_STORAGE_MESSAGE.format(foodStorage_type))
+        
+        # Prints empty space if there is no food storages to list. 
+        if not foodStorage_list:
+            print(SPACE)
+            return 
+        
+        print(listSepByTab(foodStorage_list))
+        
+        foodStorage_name = input(OPEN_FOOD_STORAGE_NAME.format(foodStorage_type))
 
+        # Handle if a food storage does not exist with this name.
+        if not self.profile.open(foodStorage_type, foodStorage_name):
+            while True:
+                newStorage_request = NEW_FOOD_STORAGE_REQUEST_MESSAGE.format(foodStorage_type) 
+                doNewStorage = input(newStorage_request)
+                if doNewStorage == YES:
+                    self.create_foodStorage(foodStorage_type)
+                    return
```

Comment:
I think you can have this return be on the same line

Reply:
Done!